### PR TITLE
Add more options/checkboxes for playing status and disabling time

### LINKED ISSUE #93: Don't show elapsed time
I would appreciate an option to disable showing the elapsed time when a track is playing.
Thank you in advance

### LINKED ISSUE #102: Add an Option to Disable the Play Indicator Icons
The play/pause/stop icons are useful when you have DiscordBee's presence running all the time, but if you have it set to only show in your status when something is being played, then the icons just sit there, cutting into the album artwork space.

Ideally, there would be a toggle as to whether or not the rich presence should use the play status indicators, or simply display the album art. Currently, you can imitate this behavior by deleting the play icons from the Discord Application's assets page, but it only works temporarily, as the images get re-uploaded again eventually.

### PR DESCRIPTION
**Pull request by VelocityRa**
The settings  window now looks like so:
image

Resolves #93.
Resolves #102.

### CODE REVIEW

**Comment by VelocityRa** [comment]
Now looks like:
image

File: SettingsWindow.Designer.cs [hunk 1]
```diff
@@ -28,363 +28,429 @@ protected override void Dispose(bool disposing)
...
-      this.label2.Location = new System.Drawing.Point(276, 66);
-      this.label2.Name = "label2";
-      this.label2.Size = new System.Drawing.Size(13, 17);
-      this.label2.TabIndex = 4;
-      this.label2.Text = "(";
-      // 
-      // textBoxState
-      // 
-      this.textBoxState.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
+            this.textBoxSmallImage = new System.Windows.Forms.TextBox();
```
Comment (sll552) [inline SettingsWindow.Designer.cs:31]: Somehow the indent on this file is wrong, please check your VS settings

**Review by sll552 — CHANGES_REQUESTED** [review]

File: SettingsWindow.Designer.cs (hunk 1 above)
Comment (VelocityRa) [reply SettingsWindow.Designer.cs]: weird

**Comment by VelocityRa** [comment]
fixed in a separate commit (tried to rebase but conflicts were annoying)

**Review by sll552 — APPROVED** [review]

**Pull request merged**